app/src/test/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewUpdateTest.kt

```diff
@@ -98,20 +100,4 @@ class BloodSugarSummaryViewUpdateTest {
             hasEffects(OpenBloodSugarUpdateSheet(bloodSugarMeasurement) as BloodSugarSummaryViewEffect)
         ))
   }
-
-  @Test
-  fun `when current facility is fetched then update the current facility`() {
```

Comment:
Why was this test deleted?

Reply:
I don't remember this. I will add it back.